src/server/scripts/Commands/cs_debug.cpp

```diff
@@ -1302,18 +1188,8 @@ class debug_commandscript : public CommandScript
         return true;
     }
 
-    static bool HandleDebugSetValueCommand(ChatHandler* handler, char const* args)
+    static bool HandleDebugSetValueCommand(ChatHandler* handler, uint32 index, Variant<uint32, float> value)
```

Comment:
I actually don't like this. It makes `.debug setvalue 0 10` and `.debug setvalue 0 10.0` do completely different things, and I think this is very much not obvious.

Reply:
Also it reminds me that I want to adjust `Variant` to use `std::variant`... might do that later

Reply:
I would argue that it's a debug command which should be used by knowledgeable people, making it an acceptable use-case. So this should really be an exception.

Additionally the command output makes it quite clear what was done (int/float is explicitly specified in the text).